app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java

```diff
@@ -184,4 +192,38 @@ public void run() {
             }
         });
     }
+
+    @SuppressLint("ResourceType")
+    public boolean howToConnectDialog(String title, String message, int iconID) {
+        if (!tvData.contains(title)) {
+            try {
+                final AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
+                LayoutInflater inflater = getLayoutInflater();
+                View dialogView = inflater.inflate(R.layout.sensor_custom_dialog, null);
+                builder.setView(dialogView);
+                builder.setTitle(title);
+                final TextView tv = (TextView) dialogView.findViewById(R.id.custom_dialog_text);
+                final ImageView iv = (ImageView) dialogView.findViewById(R.id.sensor_schematic);
+                tv.setText(String.format("Connect the %s sensor pins to the PSLab device as shown below", message));
```

Comment:
@Vikum94  Please add strings to resources file

Reply:
This string is a generated at run time and this will be used in the other dialog boxes as well. So I don't think this string should be included at the resource file @harsh-2711